tests/filtering/copydb.sh

```diff
@@ -49,16 +49,53 @@ select NULL as oid, s.restore_list_name, 'sequence owned by'
        (select 1 from source.s_table st where st.oid = s.ownedby);
 EOF
 
-pgcopydb clone --filters /usr/src/pgcopydb/exclude.ini --resume --not-consistent
+# Starts a coprocess to clone the database using the 'wal2json' plugin, applying filters from 'exclude.ini'
+coproc (pgcopydb clone --follow --plugin wal2json --filters /usr/src/pgcopydb/exclude.ini --restart)
```

Comment:
That's a little strange to me. Also we're not calling wait on the PID of the coproc to make sure to wait until the command is over with. In the other test-cases we use a separate docker service when we have to run such commands in the background, but as it's just a single one maybe using coproc is ok... as long as we then use `wait`.

Reply:
[179ff2a](https://github.com/dimitri/pgcopydb/pull/614/commits/179ff2ade7a2618c46bbbc333e563e2a83326afd)
[80fb72c](https://github.com/dimitri/pgcopydb/pull/614/commits/80fb72c660381586325ae043f1852f9957148855)